### PR TITLE
[Foodcenter] Fix various PHP Warnings and the auto refresh logic (throws a 404)

### PR DESCRIPTION
**Pull request by andygrunwald**
### What is this PR doing?

[Foodcenter] Fix various PHP Warnings and the auto refresh logic (throws a 404)

### Which issue(s) this PR fixes:

None

### Checklist

- [X] `CHANGELOG.md` entry
- [X] Documentation update

### CODE REVIEW

File: modules/foodcenter/Classes/FoodcenterPrint.php [hunk 1]
```diff
@@ -156,11 +156,8 @@ private function GetDate($time): false|string
     {
         global $func;
 
-        if ($this->config['datetime_format'] == '') {
-            return $func->unixstamp2date($time, "datetime");
-        } else {
-            return $func->unixstamp2date($time, $this->config['datetime_format']);
-        }
+        $datetimeFormat = $this->config['datetime_format'] ?? 'datetime';
+        return $func->unixstamp2date($time, "datetime");
```
Comment (M4LuZ) [inline modules/foodcenter/Classes/FoodcenterPrint.php:160]:
```suggestion
        return $func->unixstamp2date($time, $datetimeFormat);
```
I think this was your intention?
Also wouldn't `$dateTimeFormat` be the correct(erer) CamelCase naming?

**Review by M4LuZ — APPROVED** [review]
See one correction, otherwise fine to merge

File: modules/foodcenter/Classes/FoodcenterPrint.php (hunk 1 above)
Comment (andygrunwald) [inline modules/foodcenter/Classes/FoodcenterPrint.php:160]:
```suggestion
        $dateTimeFormat = $this->config['datetime_format'] ?? 'datetime';
        return $func->unixstamp2date($time, $dateTimeFormat);
```

Comment (andygrunwald) [reply modules/foodcenter/Classes/FoodcenterPrint.php]: Yes, done. Thanks.

**Pull request merged**